release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -656,6 +656,7 @@ In the following tables, features are marked with the following statuses:
 |Removed
 |Removed
 
+// Asked ALEX if status same for 4.17
```

Comment:
The SQLite format is still deprecated and has not been removed. The TLDR is that it is unlikely to ever be removed unless we have an OCP 5 release. @adellape